diff --color -Nru tezos-client-14.0-rc1.orig/build-binary.sh tezos-client-14.0-rc1/build-binary.sh
--- tezos-client-14.0-rc1.orig/build-binary.sh	2022-07-22 17:37:24.000000000 +0300
+++ tezos-client-14.0-rc1/build-binary.sh	2022-07-27 12:01:58.842885979 +0300
@@ -13,12 +13,12 @@
 
 cd tezos
 opam init local ../opam-repository --bare --disable-sandboxing
-opam switch create . --repositories=local
+opam switch create . --repositories=local --no-install
 eval "$(opam env)"
 opams=()
 while IFS=  read -r -d $'\0'; do
     opams+=("$REPLY")
-done < <(find ./vendors ./src ./tezt -name \*.opam -print0)
+done < <(find ./vendors ./src ./tezt ./opam -name \*.opam -print0)
 opam install "${opams[@]}" --deps-only --criteria="-notuptodate,-changed,-removed"
 eval "$(opam env)"
 dune build "$dune_filepath"
